deepchem/trans/transformers.py

```diff
@@ -2469,3 +2469,62 @@ def median_filter(self, size):
     image = Image.fromarray(self.Image)
     image = image.filter(ImageFilter.MedianFilter(size=size))
     return np.array(image)
+
+
+class RxnSplitTransformer(Transformer):
```

Comment:
Make sure to add a docstring here with a usage example

Reply:
yep! I will add those. The input is the reaction SMILES in the reactant>reagent>product format, and can also handle the case when there are no reagents i.e reactant>>product format.